core/transaction.py

```
@@ -0,0 +1,63 @@
+import json
+import time
+from nacl.signing import SigningKey, VerifyKey
+from nacl.encoding import HexEncoder
+from nacl.exceptions import BadSignatureError, CryptoError
+
+
+class Transaction:
+    def __init__(self, sender, receiver, amount, nonce, data=None, signature=None, timestamp=None):
```

Ruff (0.15.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)